CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Welcome to Ceph NVMe-oF Gateway contributing guide <!-- omit in toc -->
+
+## New contributor guide
+
+The Ceph NVMe-oF Gateway project pivots around 2 other major Open Source projects:
+- [SPDK](https://spdk.io/) [[source code](https://github.com/spdk/spdk/)], whih internally relies on the DPDK project.
+- [Ceph](https://ceph.io/) [[source code](https://github.com/ceph/ceph)].
+
+## Engage with the community
+
+Besides the [regular Ceph community channels](https://ceph.io/en/community/connect/), the NVMe-oF
+Gateway team can be specifically reached at:
+- [Ceph Slack workspace](https://ceph-storage.slack.com/): `#nvmeof` channel,
+- [Weekly Sync
+  Meeting](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=bzZpcWFvaG1pNnVkczZuYWdncmViZjgxaG5fMjAyMzA1MDlUMTQwMDAwWiA5dHM5YzdsdDd1MXZpYzJpanZ2cXFsZnBvMEBn&tmsrc=9ts9c7lt7u1vic2ijvvqqlfpo0%40group.calendar.google.com&scp=ALL): every Tuesday at 10:00am-10:30am (ET/EST). Past minutes and topic proposals can be found [here](https://pad.ceph.com/p/rbd_nvmeof).
```

Comment:
It feels like this link along with the spelled out time can easily go stale.  I would just refer to the pad for the schedule and the meeting details.

Reply:
I wanted to make adding the meeting to their calendars a one-click thing for contributors, since handling meetings and timezones is usually a pain. But, anyway, I'll remove that and rely on the etherpad.